### DISCUSSION TITLE
Make entry statuses NOT be specific to site

### DISCUSSION THREAD

**Discussion by matthewstick** [craftcms/cms#15076] [Ideas]
Would like an option for status, slug, title, and the entry itself to NOT be translatable in multilanguage setups. 

In our multilanguage sites, we never have entire pages that should appear in one language but not another. Aka, a page pretty much always defaults to English then some fields we want translated. 

We never have the need to add pages that only exist in one language.

Sometimes slug and title differ, sure. But not all the time. 

Structurally - we have always needed all pages to be in sync. 

What keeps happening is people accidentally get the sites out of sync. And with hundreds of potential pages. And sometimes 8 languages in one domain. It is really difficult to audit the site and see what's in sync and what isn't. 

Can you add an option to the slug, status, title and entry itself for language variants to be optional instead of mandatory?

Thanks!

**Comment by brandonkelly** [comment]
Entry types have had a “Title Translation Method” setting since Craft 3.5, and we added a “Slug Translation Method” setting in 4.5 (#8962). Both can be set to “Not translatable” if you want their values to always be consistent across all sites.

<img src="https://github.com/craftcms/cms/assets/47792/0b36a0e0-2671-4acd-99c8-c2b5fe629308" width="305" height="342" alt="An entry type’s Title Translation Method and Slug Translation Method settings">

So I’ve renamed the discussion to focus on statuses, which is the only thing here that isn’t possible yet.

**Reply by matthewstick** [reply]
Thanks for the tip. I suppose then we just modify this feature request to just non translatable status and "no splintering of entry IDs"

**Reply by brandonkelly** [reply]
I’m not sure I follow what you mean by “no splintering of entry IDs”. Each entry will only ever have a single ID, regardless of how many sites it’s enabled for. If you are seeing “an entry” get multiple IDs, you’re working with two separate entries.

**Reply by matthewstick** [reply]
Let's say a site has 100 pages and is translated to 8 languages.  That's how we think of a site.  There is never 90 pages in Spanish and 110 pages in Chinese.  There are always 100 pages.  We want the status sync'd.  We don't want sites to maintain separate documents per language.  We just use the language as a way to have different fields translated.  But what happens is slowly these sites start splintering because people forget where they are or documents get lost.  And the site needs manually audited to make sure there are always 100 documents per language.

**Reply by brandonkelly** [reply]
Sure, I get that. As long as your sections’ Propagation Method settings are set to ”Save entries to all sites enabled for this section”, all entries will exist across all sites.

**Reply by matthewstick** [reply]
When you initially make an entry yes. That is nice. Issue is they can diverge.
